### ISSUE TITLE
A11y_VisualStudioCodeClient_Customizations and settings_Changing the User settings_ Screen reader: Voiceover is not announcing the message which appears when focus comes on "Auto".

### ISSUE THREAD

**Issue by vscodenpa** [vscodenpa/testissues#10737]
--- GitHub Tags:
-A11yMAS; -A11yTCS; -A11ySev2; -Device; -Visual Studio Code Client; -BM-VisualStudioCodeClient-MAC-Oct2022; WCAG1.3.1; -Voiceover; -Screenreader; -Info and Relationships;

--- Environment and OS details:
Visual Studio Code
Screenreader: Voiceover

--- Repro Steps:

1. Open Visual studio code editor 
2. Open the command Pallet and type "setting".
3. Using down-arrow key select user setting.
4. Type the word "Auto " and select one of the suggestion given by the vs code editor.
5. Type the "Tab size" and select the option from the auto suggestion.
6. Verify that Voiceover is not announcing the message which appears when focus comes on "Auto".

--- Actual Result:
Voiceover is not announcing the message (Remove adjacent quotes ....) which appears when focus comes on "Auto".

--- Expected Result:
Voiceover should announce the message (Remove adjacent quotes ....) which appears when focus comes on "Auto".

--- Attachment:
![message](https://user-images.githubusercontent.com/98745784/197955500-494e93b0-438e-4e22-9366-8c232174be82.gif)
